crates/api_models/src/admin.rs

```diff
@@ -95,15 +95,8 @@ pub struct MerchantAccountCreate {
     #[schema(value_type = Option<Object>,example = json!({"type": "single", "data": "signifyd"}))]
     pub frm_routing_algorithm: Option<serde_json::Value>,
 
-    ///Will be used to expire client secret after certain amount of time to be supplied in seconds
-    ///(900) for 15 mins
-    #[schema(example = 900)]
-    pub intent_fulfillment_time: Option<u32>,
```

Comment:
Because of the presence of `#[serde(deny_unknown_fields)]`, this change will be a breaking change as passing these fields will throw an error. Please make required changes in the postman collection

Reply:
Have already asked the dashboard Team, they do an internal function call and not an api call to update merchant account. Also they are not passing this field.